frontend/src/App.tsx

```diff
@@ -110,7 +110,13 @@ class App extends PureComponent<Props, State> {
     this.state = {
       connectionState: ConnectionState.INITIAL,
       elements: {
-        main: fromJS([makeElementWithInfoText("Please wait...")]),
+        main: fromJS([
+          Map({
```

Comment:
The `Map` shouldn't be necessary here.

`fromJS` automatically creates a `Map` for you if you pass a pure Javascript object literal (i.e. `{something: ...}`)